code/pizzabot.js

```diff
@@ -1,7 +1,98 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+// 1. Definition of variables 
 
+const vegetarian = "Vegetarian"
+const hawaiian = "Hawaiian"
+const pepperoni = "Pepperoni"
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+
+// 2. Functions
+
+// Function for validating ordername
+const validateOrderName = (orderName) => {
+    if (orderName.toLowerCase() === vegetarian.toLowerCase()) {
+        return true;
+    } else if (orderName.toLowerCase() === hawaiian.toLowerCase()) {
+        return true;
+    } else if (orderName.toLowerCase() === pepperoni.toLowerCase()) {
+        return true;
+    } else {
+        alert('Select a pizza from the menu!');
+        return false;
+    }
+};
+
+// Function for calculating total cost
+const calculateTotalCost = (orderQuantity, pizzaPrice) => {
+    let orderTotal = orderQuantity * pizzaPrice;
+    return orderTotal;
+};
+
+// Function for calculating total cooking time
+const calculateCookingTime = (orderQuantity) => {
+    let orderTime = 0; 
+    if (orderQuantity <= 2) {
+        orderTime = 10;
+        return orderTime;
```

Comment:
Again, you don't have to create another variable if its only purpose is to be returner - you can write it like this :
```
if (orderQuantity <= 2) {
    return 10;
}
```